src/main/java/edu/ucdavis/dss/ipa/entities/LineItem.java

```diff
@@ -131,4 +131,23 @@ public String getLastModifiedBy() {
     public Date getLastModifiedOn() {
         return updatedAt;
     }
+
+    @JsonIgnore
+    @Transient
+    public String getLoginIdOfLastModifiedBy() {
+        // modifiedBy will come in the form "system" or "user:guilden"
```

Comment:
Make this a proper Javadoc comment: what args it takes, what it returns, and this explanation of the forms modifiedBy will take.